### ISSUE TITLE
Line Ending changes ignored in diff editor

### ISSUE THREAD

**Issue by asymetrixs** [Microsoft/vscode#63025]
- VSCode Version: 1.28.1 x64
- OS Version: Windows 7 Enterprise

Steps to Reproduce:

1. Make sure you open a folder in VScode that is a git repository with git-scm installed
2. Open a file with line ending LF
3. Modify a line in that file (edit, delete, add, does not matter)
4. Change the line ending to CRLF
5. Save the file
6. Go to 'Source Control' and click on the modified file
7. An editor opens showing the changes in that file between the original and the modified version
8. Add a commit message and commit the changes

Problem:
In step 7 the editor does not show that the line endings were modified. After step 8, doing a diff on the original file and the new file the line ending changes become visible, but then it is already too late.
This is especially annoying when working on Windows and Unix. Moreover the real diff becomes unreadable, because basically it shows the whole file has been modified.
Additionally, when clicking in step 7 in one of the files, the cursor position at the bottom status line is updated, but the line ending indicator shows only the line ending format of the new file (for both). Not the original one. So it is impossible to see this change.
Either the line endings should be highlighted in the editor or at least the background of the line ending indicator in the status bar should change to red or something that grabs attention.

Does this issue occur when all extensions are disabled?: Yes

**Comment by alexandrudima** [comment]
Duplicate of #19254
